conf/application.conf

```diff
@@ -109,3 +109,8 @@ stride.roles {
 
 internal-auth.token = "YWdlbnQtYXNzdXJhbmNl" ##base64: agent_assurance
 internal-auth-token-enabled-on-start = false
+
+agent.cache.enabled = true
+agent.cache.expires = 4 hours
```

Comment:
default should be 15 mins...if an agent is suspended then there should be at most 15 min delay before it takes effect.